src/main/java/org/folio/service/CirculationSettingsService.java

```diff
@@ -42,10 +45,10 @@ public Future<Response> getAll(int offset, int limit, String query) {
       GetCirculationSettingsStorageCirculationSettingsResponse.class);
   }
 
-  public Future<Response> create(CirculationSetting circulationSetting) {
-    return PgUtil.post(CIRCULATION_SETTINGS_TABLE, circulationSetting, okapiHeaders, vertxContext,
-      PostCirculationSettingsStorageCirculationSettingsResponse.class)
-      .compose(eventPublisher.publishCreated());
+  public Future<CirculationSetting> create(CirculationSetting circulationSetting) {
+    return repository.save(circulationSetting.getId(), circulationSetting)
+      .map(circulationSetting)
```

Comment:
Can we change `AbstractRepository#save` from `postgresClient.save(tableName, id, entity)` to `postgresClient.saveAndReturnUpdatedEntity(tableName, id, entity)`? This way we won't have to do this mapping.

Reply:
done